### ISSUE TITLE
Accessibility for Scalar API Reference

### ISSUE THREAD

**Issue by kayla-beckham** [scalar/scalar#4572]
### What happens?

There are just a few more undocumented accessibility issues from our audit we wanted to communicate with you.

First, for the keyboard navigation of the Test Request API Client feature, our testers expect that when the Test Request pane is opened, the keyboard focus should move to the first interactive element in the dialog. This would be the button that opens/closes the sidebar:
![Image](https://github.com/user-attachments/assets/74b082b2-3cd0-4dc8-8206-9dec2c3e6dd6)

The next two issues are around the 'Client Libraries' list in the introduction section:

![Image](https://github.com/user-attachments/assets/753ecb52-3cbf-485d-a55b-3b0a3237c46b)

For these tabs, certain roles and accessibility attributes are expected. 
- role="tablist" for tab container
- role="tab" for tab
- role="tabpanel" for page under the tab
- aria-control - for each tab referring to associated tabpanel
- aria-select - true for the current page, and false for others

And regarding screenreader accessibility, the label 'Client Libraries' is not announced when the tabs under it receive focus.

### What did you expect to happen?

When a dialog opens, the keyboard focus moves to the first interactive element.

The 'Client Libraries' tabs have the expected roles and accessibility attributes for that type of component.

The group label for the 'Client Libraries' tabs is announced by a screen reader when a tab receives focus.

**Comment by antlio** [comment]
thank you so much @kayla-beckham! we'll look at those asap!
